R/sun-methods.R

```diff
@@ -76,13 +109,31 @@ setMethod("crepuscule", signature(crds="SpatialPoints", dateTime="POSIXct"),
                          direction=direction, POSIXct.out=POSIXct.out)
           })
 
-###_ + sunriset methods
-if (!isGeneric("sunriset")) {
-    setGeneric("sunriset", function(crds, dateTime, ...) {
-        standardGeneric("sunriset")
-    })
-}
+#' @title Calculate sunrise/sunset
+#' @description Calculates sunrise or sunset at a specific geographical location and time.
+#' depending on the "direction" parameter. Methods are available
```

Comment:
when referring to parameters or arguments, use backticks like `direction` - which renders them in courier font